src/Microsoft.OData.Core/MultipartMixed/ODataMultipartMixedBatchReader.cs

```diff
@@ -0,0 +1,466 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataMultipartMixedBatchReader.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.MultipartMixed
+{
+    #region Namespaces
+    using System;
+    using System.Diagnostics;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Text;
+
+#if ODATALIB_ASYNC

```

Comment:
Should this be #if PORTABLELIB?

Reply:
Yes, will fix.

Reply:
fixed.

---
In reply to: [140611491](https://github.com/OData/odata.net/pull/960#discussion_r140611491) [](ancestors = 140611491)